Automattic/wp-calypso
# Fix typos in Gutenboarding plans

#### Changes proposed in this Pull Request

In the Gutenboarding plans data, fix the following issues

* Typo: Change `Remove WordPress ads` to `Remove WordPress.com ads`
* Typo: Fix capitalisation in `Remove Wordpress.com ads` in details section to `Remove WordPress.com ads`
* Separate `Upload themesRemove WordPress.com branding` into two different rows
* Rename the `Commerce` plan column to `eCommerce` to match the plan name

Note: some of the wording has been changed slightly to better align with `wordpress.com/pricing`, while still trying to keep true to the designs we worked from.

#### Testing instructions



From `/new` click the plans button at the top right, and click More Details to expand the full plan details.

* Check that the typos are fixed, and make sure the check marks against the plans match what is at `wordpress.com/pricing` (if you can't see full list of plans there, try `wordpress.com/pricing/?currency=USD`)

Fixes #
